OneSignalSDK/onesignal/notifications/src/main/java/com/onesignal/notifications/internal/lifecycle/impl/NotificationLifecycleService.kt

```diff
@@ -97,7 +97,7 @@ internal class NotificationLifecycleService(
         data: JSONArray,
         notificationId: String,
```

Comment:
This `notificationId` is now unused so let's remove it. (Also maybe we can remove it in more places up the stack too)